CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog for Gradle Jacoco Coverage Plugin
+
+## 0.0.1
```

Comment:
I think 0.1.0 might make sense for an initial release (if I'm understanding semver properly)